packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
I see. My understanding from the [default-iife-1128](https://github.com/babel/babel/blob/master/packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-iife-1128/exec.js) test is that default parameters are scoped to the parent, not the function itself. Is it different when the referenced identifier is another parameter?

Here's a [repl example](http://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3&experimental=false&loose=true&spec=true&code=const%20bar%20%3D%20true%3B%0A%0Afunction%20foo(a%20%3D%20bar%2C%20...b)%20%7B%0A%20%20const%20bar%20%3D%20false%3B%0A%20%20assert.equal(a%2C%20true)%3B%0A%20%20assert.equal(b%5B0%5D%2C%202)%3B%0A%20%20assert.equal(b%5B1%5D%2C%203)%3B%0A%7D%0A%0Afunction%20foo(a%20%3D%20bar%2C%20...b)%20%7B%0A%20%20function%20deopt()%20%7B%20b%3B%20%7D%3B%0A%20%20const%20bar%20%3D%20false%3B%0A%20%20assert.equal(a%2C%20true)%3B%0A%20%20assert.equal(b%5B0%5D%2C%202)%3B%0A%20%20assert.equal(b%5B1%5D%2C%203)%3B%0A%7D).